foundation_cms/legacy_apps/mozfest/templates/mozfest/mozfest_homepage.html

```diff
@@ -12,7 +12,7 @@
 {% block bodyclass %}mozfest {% if banner_video_type %}banner-type-{{ banner_video_type }}{% endif %}{% endblock bodyclass %}
 
 {% block hero_guts %}
-    {% include "partials/primary_hero.html" with page=page %}
+    {% include "partials/primary_hero.html" with page=page request=request %}
```

Comment:
This update was made as an idea to try and fix the "request not found" issue I was experiencing when trying to render the block in the `primary_hero` template